packages/CodeGenLib/src/Database/manage-metadata.ts

```diff
@@ -221,6 +320,8 @@ export class ManageMetadataBase {
                                     Scale=${veField.Scale}
                                   WHERE 
                                     ID = '${field.ID}'`; // don't need to update the __mj_UpdatedAt field here, that happens automatically via the trigger
+               
```

Comment:
@JS-BC  same comment here and everyehwere as above to do it with a wrapper function.